macOS/DuckDuckGo/Preferences/Model/TabsPreferences.swift

```diff
@@ -35,7 +35,7 @@ struct TabsPreferencesUserDefaultsPersistor: TabsPreferencesPersistor {
     @UserDefaultsWrapper(key: .newTabPosition, defaultValue: .atEnd)
     var newTabPosition: NewTabPosition
 
-    @UserDefaultsWrapper(key: .sharedPinnedTabs, defaultValue: false)
+    @UserDefaultsWrapper(key: .sharedPinnedTabs, defaultValue: true)
```

Comment:
Instead of default value set to `false` and enabling it for existing users, I’ve decided to default it to `true` and disable it for new users. This ensures that existing users don’t end up in a state where shared pinned tabs are active, but the value is set to `false`.